proxies/live/apiproxy/proxies/default.xml

```diff
@@ -40,7 +40,7 @@
       <Description/>
       <Request>
         <Step>
-          <Name>KeyValueMapOperations.GetSecureVariables</Name>
+          <Name>KeyValueMapOperations.GetSharedSecureVariables</Name>
```

Comment:
This needs to be an _additional_ step; not a replacement step. Otherwise we won't have access to `private.apigee.NHSD-ASID`.

Reply:
added in as an extra step